src/lib/lang-runner-java.ts

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as os from 'os';
+import * as path from 'path';
+import * as fs from 'fs';
+import * as https from 'https';
+import * as execa from 'execa';
+import LangRunner from '../lib/lang-runner';
+import LocalRun from '../lib/local-run';
+
+const runtimeJarName = 'sf-fx-runtime-java-runtime-1.0.3-jar-with-dependencies.jar';
+const runtimeJarUrl = `https://repo1.maven.org/maven2/com/salesforce/functions/sf-fx-runtime-java-runtime/1.0.3/${runtimeJarName}`;
```

Comment:
Nitpick: Since the version is both in the file name _and_ the path, it might make sense to introduce a new variable with the version and use it both in `runtimeJarName` and `runtimeJarUrl` instead of a hardcoded `1.0.3` in multiple locations.

Reply:
I agree. Fixed in 43f5c9d90d0122ed1997fbc83dff30e90e2acd64.